hw2/task05/hw2_task05.py

```diff
@@ -0,0 +1,31 @@
+"""Custom range.
+
+Some of the functions have a bit cumbersome behavior when we deal with
+positional and keyword arguments.
+
+Write a function that accept any iterable of unique values and then
+it behaves as range function:
+import string
+assert = custom_range(string.ascii_lowercase, 'g') == ['a', 'b', 'c', 'd', 'e', 'f']
+assert = custom_range(string.ascii_lowercase, 'g', 'p') == ['g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o']
+assert = custom_range(string.ascii_lowercase, 'p', 'g', -2) == ['p', 'n', 'l', 'j', 'h']
+"""
+
+
+def custom_range(
+    it_obj: str, start: str = None, end: str = None, step: int = 1
```

Comment:
Please follow the description, not by example.  `any iterable of unique values`  is not only str